src/INode.java

```diff
@@ -427,14 +450,12 @@ private ASTValue evalRemove(WordsEnvironment environment) {
 		throw new AssertionError("Not yet implemented");
 	}
 
-	private ASTValue evalRepeat(WordsEnvironment environment) {
+	private ASTValue evalRepeat(WordsEnvironment environment) throws WordsException {
 		ASTValue times = children.get(0).eval(environment);
 		AST statementList = children.get(1);
 		
 		if (times.type != ValueType.NUM) {
-			// TODO
-			// throw exception
-			return null;
+			throw new InvalidTypeException(this.lineNo, ValueType.NUM.toString(), times.type.toString());
```

Comment:
My original code here was also inconsistent with the reference manual.  We allow a string if it can be coerced to a number.  You could either fix or open an issue for that and assign it to me.